pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,117 @@
+package bundlesecret
+
+//export OPENSHIFTCONFIG=./openshift.local.certificates/admin/.kubeconfig
+import (
+	"encoding/base64"
+	"encoding/binary"
+	"fmt"
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// MaxFileSize?
+const MaxSecretSize = 1 * 1024 * 1024
+const MaxFileSize = 1 * 1024 * 1024
+
+func NewCmdBundleSecret(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+	var sourcelist util.StringList
+	cmd := &cobra.Command{
+		Use:   "bundle-secret NAME -f SOURCENAME",
+		Short: "bundle data from a source list to load kubernetes secret",
+		Long:  "bundle data from a source list to load kubernetes secret",
+		Run: func(cmd *cobra.Command, args []string) {
+			var err error
+			if len(args) == 0 {
+				err = fmt.Errorf("Must specify NAME as an argument")
+				checkErr(err)
+			}
+			objname := args[0]
+			if len(sourcelist) == 0 {
+				err = fmt.Errorf("Must specify files and/or directories to gather data from.")
+				checkErr(err)
+			}
+			for _, isspace := range sourcelist {
+				if isspace == "" {
+					err = fmt.Errorf("No spaces allowed.  Separate source names with commas, no spaces.")
```

Comment:
@liggitt passing the file / folder list as an argument works, but then we need to provide a name for the secret itself.  Where else do we use -f as the target to write output so this would not be inconsistent?  How would you want to provide the name for the secret?